### PR TITLE
Expanded README file

### PR DESCRIPTION
**Pull request by danMateer**
Just added some bs stuff.

Let me know what you think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# intro
+# Intro
 
 Into
-Hi Dan
+Hi Sydney!
```
Comment (sydneyblom) [inline README.md]: Hi

File: README.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+
+Here's where we'd describe the purpose of this repository.
+
+Also here's a math problem: 8 * 8 * 9 = ?
```
Comment (sydneyblom) [inline README.md]: No

File: README.md (hunk 1 above)
Comment (danMateer) [reply README.md]: Hello again, thanks for the feedback.

File: README.md (hunk 2 above)
Comment (danMateer) [reply README.md]: Agreed.

**Pull request merged**